bloom-housing/bloom
# fix: prevent listings without AMI charts

This PR addresses [#692](https://github.com/metrotranscom/doorway/issues/692)


## Description

It's a change from task comment. It will throw an error when trying to add unit without ami chart

## How Can This Be Tested/Reviewed?

Create new unit without touching `AMI Chart` dropdown.
Try to save -> It should throw validation error (prevent from saving)

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
